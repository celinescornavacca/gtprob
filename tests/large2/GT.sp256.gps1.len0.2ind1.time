10.19user 0.17system 0:10.60elapsed 97%CPU (0avgtext+0avgdata 78416maxresident)k
156inputs+0outputs (0major+54961minor)pagefaults 0swaps
