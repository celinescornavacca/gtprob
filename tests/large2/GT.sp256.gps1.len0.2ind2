(((((((((HF0,(GJ0,HJ0)),BD0),(AB0,BB0)),PA0),(((BJ0,CJ0),VB0),(WH0,XH0))),(((ED0,(MJ0,NJ0)),(RA0,(JC0,(GD0,(PG0,(DH0,EH0)))))),((HA0,(FC0,(((DF0,(ZF0,AG0)),(ME0,(QE0,(QF0,RF0)))),(SJ0,TJ0)))),(((QG0,RG0),(((NE0,(VG0,WG0)),LE0),TC0)),((DG0,EG0),SF0))))),((CA0,(((XA0,YA0),(XF0,YF0)),IA0)),((((QA0,(ZD0,AE0)),(SC0,((XG0,YG0),(JE0,KE0)))),((SD0,(VH0,(EI0,FI0))),WA0)),((SG0,TG0),IE0)))),(((((((KC0,((OJ0,PJ0),GG0)),(((QI0,RI0),PD0),((BG0,CG0),DD0))),(((FD0,(ZH0,(CI0,DI0))),(ZA0,((FF0,(NG0,OG0)),(CE0,DE0)))),((((IC0,(((FJ0,(QJ0,RJ0)),(OF0,PF0)),(IF0,JF0))),(MF0,NF0)),(QC0,(VD0,(HG0,IG0)))),((((JH0,KH0),(BC0,CC0)),(SA0,(LB0,(FG0,(SI0,(KJ0,LJ0)))))),(MB0,NB0))))),(FA0,(UC0,VC0))),(DA0,(TA0,UA0))),((((AA0,((((((RC0,((YH0,(VI0,WI0)),(BH0,CH0))),EC0),((((DJ0,EJ0),XE0),((IJ0,JJ0),JI0)),OA0)),(MD0,(JG0,KG0))),(MA0,((KD0,LD0),HD0))),(RE0,SE0))),((TH0,UH0),UG0)),(((GE0,HE0),AD0),BA0)),(((((MC0,NC0),TB0),(KA0,LA0)),EA0),((CD0,(BF0,CF0)),(AI0,BI0))))),((((((GC0,HC0),(CB0,DB0)),(((((EE0,FE0),((MI0,NI0),(XI0,YI0))),((HB0,(HH0,IH0)),GB0)),(EB0,((OI0,PI0),((TD0,UD0),(VE0,WE0))))),(YE0,(GF0,(TF0,UF0))))),(((((FH0,GH0),(OB0,PB0)),IB0),((WC0,XC0),(LH0,MH0))),((ND0,OD0),JD0))),(((DC0,(KI0,LI0)),(((NH0,OH0),(OE0,PE0)),(((LG0,MG0),(ZE0,AF0)),SB0))),((GI0,HI0),(TE0,UE0)))),(((OC0,PC0),NA0),(((((YC0,ZC0),KB0),((BE0,((ZG0,AH0),EF0)),((ZB0,AC0),(XB0,YB0)))),(JA0,((LC0,((ZI0,AJ0),II0)),(WD0,(XD0,YD0))))),(((((RH0,SH0),(WB0,(KF0,LF0))),JB0),VA0),((QB0,RB0),FB0))))))),((((((PH0,QH0),(TI0,UI0)),UB0),(VF0,WF0)),GA0),(((QD0,RD0),ID0),(UJ0,VJ0))));
