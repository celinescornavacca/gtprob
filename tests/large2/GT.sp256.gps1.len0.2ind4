(((((CC0,DC0),((AE0,((YG0,ZG0),EF0)),SD0)),(VF0,WF0)),((((GJ0,HJ0),GH0),((JI0,KI0),ZD0)),(RC0,(CF0,DF0)))),(((((SE0,TE0),EB0),(((FA0,((LB0,((LI0,MI0),AD0)),(XB0,(WE0,XE0)))),(((WC0,(KJ0,LJ0)),CB0),BB0)),((((ZE0,AF0),(TD0,(((AJ0,BJ0),VI0),GE0))),((NB0,OB0),(QA0,(OF0,PF0)))),(((EG0,FG0),(XC0,(TF0,UF0))),(NA0,OA0))))),((((FB0,((CE0,DE0),(WI0,XI0))),(KE0,((NG0,OG0),JF0))),((IB0,(KD0,(RG0,SG0))),(HA0,IA0))),((ED0,FD0),(UA0,VA0)))),(((JA0,(WA0,((ZC0,(MF0,NF0)),AB0))),(LA0,MA0)),((((((((DI0,EI0),(OJ0,PJ0)),(PB0,((FI0,GI0),LH0))),((((((NC0,OC0),DB0),(LC0,MC0)),(((WD0,XD0),(QD0,RD0)),(SJ0,TJ0))),GA0),(KA0,(UD0,VD0)))),AA0),(((((EA0,((((MB0,((EH0,FH0),RB0)),KB0),(((((IC0,(BF0,(WH0,(MJ0,NJ0)))),SB0),((QE0,RE0),JD0)),(UG0,VG0)),ZA0)),((BI0,CI0),(VB0,WB0)))),((OD0,PD0),DA0)),(((UB0,(((FF0,GF0),(XF0,YF0)),JC0)),(OE0,PE0)),(HE0,IE0))),BA0),((((DD0,((SF0,(PG0,QG0)),KF0)),(GD0,HD0)),(LD0,MD0)),(UE0,VE0)))),((((RA0,((((CJ0,DJ0),YC0),(((UJ0,VJ0),(XH0,YH0)),UC0)),GB0)),(((EC0,FC0),JB0),((WG0,XG0),((IJ0,JJ0),KH0)))),(((LE0,(IH0,JH0)),((NI0,OI0),(QF0,RF0))),(((AH0,BH0),(QJ0,RJ0)),((EE0,FE0),(SA0,((HI0,II0),((AG0,BG0),JE0))))))),((XA0,((MG0,(SH0,TH0)),(JG0,((ZH0,AI0),LG0)))),(((BD0,CD0),YA0),(((YE0,(IG0,(TI0,UI0))),ND0),(LF0,((RI0,SI0),TG0))))))),(((YB0,ZB0),(PA0,((YD0,(KG0,(PI0,QI0))),HB0))),((((HF0,IF0),((((TB0,(SC0,TC0)),(UH0,VH0)),(((VC0,(MH0,NH0)),(QB0,(ME0,NE0))),((((QH0,RH0),HH0),KC0),(AC0,BC0)))),(((CH0,DH0),BE0),(PC0,QC0)))),((ID0,((YI0,ZI0),ZF0)),(GG0,HG0))),(CA0,(((((GC0,HC0),(CG0,DG0)),TA0),(EJ0,FJ0)),(OH0,PH0)))))))));
