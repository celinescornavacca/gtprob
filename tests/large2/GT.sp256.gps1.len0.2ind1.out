-364.749686134
